tests/errors/semantic/blocking/INSERT_LIST_INCOMPATIBLE_TYPES_2.py

```diff
@@ -0,0 +1,6 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring
+
+a = [1,2,3]
+b = (4,5)
+a.append(3, b)
```

Comment:
This is a call to `append`, not `insert`.

Reply:
fixed in cfa5a2a986d745a91d730022bb3f166c06a32311